.github/labeler.yml

```diff
@@ -1,22 +1,24 @@
-"frontend":
-    - src/contour/*.cpp
-    - src/contour/*.h
+"CI":
+    - .github/**
+"CMake":
+    - cmake/**
```

Comment:
Here too, have `CmakeLists.txt` added. Does this make sense?

Reply:
Let's try `**CMakeLists.txt`.